### DISCUSSION TITLE
Will the JavaScript code translated by Fusion work on Rhino?

### DISCUSSION THREAD

**Discussion by ghost** [pfusik/cito#118] [Q&A] [answered]
https://github.com/mozilla/rhino

**Comment by pfusik** [comment, marked as answer]
[One of my Fusion projects](https://asap.sourceforge.net) worked in Rhino in 2010. I'm not up-to-date with Rhino. A quick look at its [compatibility table](https://mozilla.github.io/rhino/compat/engines.html) reveals that it will probably won't work at all because it doesn't support `class`.
